src/pages/selfStudyCheck/index.tsx

```
@@ -43,8 +44,18 @@ export const SelfStudyCheck = () => {
       }
     >
       <Content>
+        <TitleContainer>
+          <PeriodText>목요일</PeriodText>
+          <PeriodMap>
+            {period.map((item) => (
+              <PeriodText>{item}</PeriodText>
+            ))}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)